addPost.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    require_once 'src/Models/BlogModel.php';
+    require_once 'src/connectToDb.php';
+
+    session_start();
+    $db = connectToDb();
+
+    if (!isset($_SESSION['userid'])) {
+        header('Location: src/login.php');
```

Comment:
should the login.php file be in the src folder? isnt it a page?

Reply:
Yep - amended